src/utils/line.ts

```diff
@@ -407,19 +400,16 @@ export const formatSecuredLineData = (
             token: _createTokenView(
               token,
               BigNumber.from(totalVolume),
-              BigNumber.from(totalVolumeUsd).div(BigNumber.from(totalVolume))
-            ), // use avg price at time of revenue
-            // amount: totalVolume,
-            // value: (totalVolumeUsd ?? '0').toString(),
-            amount: totalRevenueVolume,
-            value: formatUnits(usdcPrice.mul(unnullify(totalRevenueVolume).toString()), 6).toString(),
+              BigNumber.from('0') // TODO: // use avg price at time of revenue
```

Comment:
totalVolumeUsd is already priced at the time of revenue so im pretty sure that makes it the "avg price at time of revenue" (although its time of revenue claimed in spigot, not earned on protocol but no one cares)

Reply:
needs to fix the subgraph before can add this back in: https://www.notion.so/debtdao/Subgraph-spigot-summary-field-totalVolumeUsd-is-incorrect-6273f36f2b4540038a207aeb0748daf9?pvs=4